src/libcore/ptr/const_ptr.rs

```diff
@@ -295,23 +295,26 @@ impl<T: ?Sized> *const T {
         intrinsics::ptr_offset_from(self, origin)
     }
 
-    /// Returns whether two pointers are guaranteed equal. Returns `true` whenever the
-    /// pointers are known to be equal.
+    /// Returns whether two pointers are guaranteed equal.
+    /// At runtime this function behaves like `self == other`.
     /// In some contexts (e.g., compile-time evaluation),
```

Comment:
```suggestion
    /// Returns whether two pointers are guaranteed equal.
    /// 
    /// At runtime this function behaves like `self == other`.
    /// However, in some contexts (e.g., compile-time evaluation),
```
Yeah now I am nitpicking. ;)